cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -113,6 +113,9 @@
 
 @SuppressWarnings("checkstyle:ClassDataAbstractionCoupling")
 public class KafkaCluster extends AbstractModel {
+    // Name of component kafka
```

Comment:
I find it a bit weird that out of the 12 places where this is defined the comment is only here ;-)

Reply:
Embarassing - originally only has these for kafka and zookeeper then saw the value in applying to them all.

Reply:
No worries.